Detectors/MUON/MCH/Simulation/include/MCHSimulation/MCHDigitizer.h

```diff
@@ -0,0 +1,80 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/** @file MCHDigitizer.h
+ * C++  MCH Digitizer.
+ * @author Michael Winn, Laurent Aphecetche
+ */
+
+#ifndef O2_MCH_SIMULATION_MCHDIGITIZER_H_
+#define O2_MCH_SIMULATION_MCHDIGITIZER_H_
+
+#include "MCHSimulation/Digit.h"
+#include "MCHSimulation/Detector.h"
+#include "MCHSimulation/Geometry.h"
+#include "MCHSimulation/Hit.h"
+#include "MCHSimulation/Response.h"
+#include "MCHMappingInterface/Segmentation.h"
+
+#include "TGeoManager.h"
+
+namespace o2
+{
+namespace mch
+{
+
+class MCHDigitizer
```

Comment:
In the end I would drop the MCH prefix of the class (and file) name. Looks like we are the only one using a prefix (which feels a bit like stutter given the namespace : mch::MCHDigitizer ... )

Reply:
Ok, done. Sandro was slightly in favour of having it, but I removed it now.